mathics/builtin/__init__.py

```diff
@@ -116,22 +116,25 @@ def is_builtin(var):
 
 
 # FIXME: redo using importlib since that is probably less fragile.
-exclude_files = set(("files", "codetables", "base", "importexport", "colors"))
+exclude_files = set(("codetables", "base"))
 module_names = [
     f for f in __py_files__ if re.match("^[a-z0-9]+$", f) if f not in exclude_files
 ]
 
-if ENABLE_FILES_MODULE:
-    module_names += ["files", "importexport"]
-
 modules = []
 import_builtins(module_names)
 
 _builtins = []
 builtins_by_module = {}
 
-for subdir in ("drawing", "numbers", "specialfns",):
+disable_file_module_names = ["files_io.files", "files_io.importexport"] if ENABLE_FILES_MODULE else []
```

Comment:
I do not get this line: why you put things in a list of disabled, if that modules are enabled?

Reply:
You are absolutely correct - this is backwards! Fixing now...

(It was an artifact of not properly transforming more complicated code...)